packages/core/admin/admin/src/translations/sk.json

```diff
@@ -92,51 +164,72 @@
   "Settings.roles.list.button.add": "Pridať novú rolu",
   "Settings.roles.list.description": "Zoznam rolí",
   "Settings.roles.title.singular": "Rola",
+  "Settings.sso.description": "Upravte nastavenia pre Single Sign-On.",
+  "Settings.sso.form.defaultRole.description": "Prepojí nového autentifikovaného používateľa s vybranou rolou",
+  "Settings.sso.form.defaultRole.description-not-allowed": "Pre čítanie admin rolí potrebujete mať potrebné oprávnenia",
+  "Settings.sso.form.defaultRole.label": "Predvolená rola",
+  "Settings.sso.form.registration.description": "Vytvoriť nového používateľa pri SSO prihlásení, ak taký účet neexistuje",
+  "Settings.sso.form.registration.label": "Auto-registrácia",
+  "Settings.sso.title": "Single Sign-On",
   "Settings.webhooks.create": "Vytvoriť Webhook",
-  "Settings.webhooks.create.header": "Vytvoriť nový header",
+  "Settings.webhooks.create.header": "Vytvoriť novú hlavičku",
   "Settings.webhooks.created": "Webhook bol vytvorený",
   "Settings.webhooks.event.publish-tooltip": "Táto udalosť existuje iba pre obsah s povoleným Draft/Publish systémom",
   "Settings.webhooks.events.create": "Vytvoriť",
   "Settings.webhooks.events.update": "Upraviť",
   "Settings.webhooks.form.events": "Eventy",
-  "Settings.webhooks.form.headers": "Headery",
+  "Settings.webhooks.form.headers": "Hlavičky",
   "Settings.webhooks.form.url": "Url",
+  "Settings.webhooks.headers.remove": "Odstrániť riadok hlavičky č. {number}",
   "Settings.webhooks.key": "Klúč",
   "Settings.webhooks.list.button.add": "Pridať nový Webhook",
   "Settings.webhooks.list.description": "Získajte upozornenia na POST request zmeny.",
   "Settings.webhooks.list.empty.description": "Pridajte Váš prvý webhook do zoznamu.",
   "Settings.webhooks.list.empty.link": "Pozrieť si našu dokumentáciu",
   "Settings.webhooks.list.empty.title": "Zatiaľ neexistujú žiadne Webhooky",
+  "Settings.webhooks.list.th.actions": "akcie",
+  "Settings.webhooks.list.th.status": "stav",
   "Settings.webhooks.singular": "Webhook",
   "Settings.webhooks.title": "Webhooky",
-  "Settings.webhooks.trigger": "Trigger",
-  "Settings.webhooks.trigger.cancel": "Zrušiť Trigger",
+  "Settings.webhooks.to.delete": "{webhooksToDeleteLength, plural, one {# vybraný záznam} few {# vybrané záznamy} other {# vybraných záznamov}}",
```

Comment:
### Question:
How do we specify plural translations when number is between 2 and 4? I used keyword `few` as I know it from other translation libraries, but I'm not sure, if it is supported here.

The way how it works in slovak language is, that there exists a form of noun or adjective for:
`1` entity
`2 to 4` entities
`5 and more` entities
(`0` is the same as `5 and more`)

Reply:
hi! 
I checked the [documentation](https://formatjs.io/docs/core-concepts/icu-syntax/#selectordinal-format) of the library we use and it supports few, and it adapts the range to the language - I also tested your branch just to be sure:
<img width="644" alt="image" src="https://user-images.githubusercontent.com/71838159/207075540-c8b863d6-f80f-411e-b749-2688b68d0dc6.png">
<img width="642" alt="image" src="https://user-images.githubusercontent.com/71838159/207075594-5aca1eac-cbe6-4750-ad5f-fc0e1cc99b44.png">
<img width="786" alt="image" src="https://user-images.githubusercontent.com/71838159/207075645-1d43f556-ad90-40a5-abf6-0357cf68451f.png">
<img width="989" alt="image" src="https://user-images.githubusercontent.com/71838159/207075686-f4e5a03b-9b99-4add-b62f-82eea1dc32c9.png">

I didn't know about those specificities, glad to know more now 😄

Reply:
Thanks!